okonomiyaki/platforms/tests/test_epd_platform.py

```diff
@@ -24,6 +26,13 @@
 
 
 class TestEPDPlatform(unittest.TestCase):
+    def setUp(self):
+        self.platform_strings = tuple(
```

Comment:
I guess it's being consumed by `EPDPlatform` and we only care about the output of that?